inform7/extensions/basic_inform/Sections/Glulx and Glk.w

```diff
@@ -119,40 +117,71 @@ Glk events can be handled with the glk event handling rules.
 =
 Chapter - Glk events
 
-The glk event handling rules is a glk event based rulebook.
-The glk event handling rules is accessible to Inter as "GLK_EVENT_HANDLING_RB".
+To decide what glk event is (evtype - glk event type) glk event:
+	(- GLK_EVENT_TY_New({-new: glk event}, {evtype}) -).
 
-The glk event window is a glk window variable.
-The glk event window variable translates into Inter as "Glk_Event_Struct_win".
-The glk event value 1 is a number variable.
-The glk event value 1 variable translates into Inter as "Glk_Event_Struct_val1".
-The glk event value 2 is a number variable.
-The glk event value 2 variable translates into Inter as "Glk_Event_Struct_val2".
+To decide what glk event is a/-- character event for/of/with (C - unicode character):
```

Comment:
I felt like I was doing something wrong adding `a/--` to all of these phrases, but a natural use like `process a character event for the special character` won't compile without it. And I can see that other phrases also add a `a/--`.

It would be nice if articles were just ignored outside of `say` phrases. Or even in all of them, if two phrases aren't distinguished by having different articles.